book/src/integrate/ios_headers.md

```diff
@@ -58,8 +58,9 @@ Finally, use `dummy_method_to_enforce_bundling` somewhere within
 
 If there are [multi-blocks](https://cjycode.com/flutter_rust_bridge/feature/multiple_files.html):
 
-- For iOS, just add the 1st generated block-header files in `Runner-Bridging-Header.h`.
-- For MacOS, just add the 1st generated block-header files as **Objective-C Bridging Header**.
+- For iOS, just add the shared C header file(`#import "bridge_generated_shares.h"` as default) along with `#import "GeneratedPluginRegistrant.h"` in `Runner-Bridging-Header.h`. And there is no need for other C header files.
```

Comment:
nit: not very sure, shall we call it bridge_generated_shares, or bridge_generated_share, or bridge_generated_shared?

Reply:
After some thought, also I asked ChatGpt, and it turns out that `shared` is clearer than the other 2. Also, I extract the name into a constant global var in [8284409](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145/commits/82844095b64559dbfac828c9540ea269f1ca16f4).